deploy/examples/operator-openshift.yaml

```diff
@@ -632,6 +632,12 @@ data:
   #       requests:
   #         cpu: 200m
   #         memory: 128Mi
+
+  # (Optional) Burst to use while communicating with the kubernetes apiserver.
+  # CSI_KUBE_API_BURST:
```

Comment:
```suggestion
  # CSI_KUBE_API_BURST: "10"
```

Reply:
add some values for the operator-openshift.yaml and operator.yaml so that user will know that they need to set the int values